File: Two-Sum.py
1class Solution:
2    def twoSum(self, nums: List[int], target: int) -> List[int]:
3        p={}
4        for i in range(len(nums)):
5            if target-nums[i] in p.keys():
6                return i,p[target-nums[i]]
7            p[nums[i]]=i
8
9        

File: 1-two-sum/two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        HashMap<Integer,Integer> h = new HashMap<>();
        for(int i=0;i<nums.length;i++){
            if(!h.containsKey(nums[i])){
                h.put(target-nums[i],i);
            }
            else return new int[] {i,h.get(nums[i])};
        }
        return new int[]{};
    }
}